src/OrchardCore.Modules/OrchardCore.Workflows/OrchardCore.Workflows.csproj

```diff
@@ -26,6 +26,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Data.Abstractions\OrchardCore.Data.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Data.YesSql\OrchardCore.Data.YesSql.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Deployment.Abstractions\OrchardCore.Deployment.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Deployment.Core\OrchardCore.Deployment.Core.csproj" />
```

Comment:
Why the OC.Deployement.Abstractions & Core are both exists?

Reply:
The `TemporaryFileBuilder` class is from `OrchardCore.Deployment.Core` module